### DISCUSSION TITLE
How to setup cors globally for Resource Routes?

### DISCUSSION THREAD

**Discussion by wong2** [sergiodxa/remix-utils#375] [Q&A]
I have several resource routes with the prefix `/api`, which are called by both my Remix app and a browser extension. Now I want to fix CORS requests for the browser extension.
However, the `cors` function doesn't work with it, after some debugging, I found that the `handleRequest` function isn't even called when accessing these resource routes.

**Comment by sergiodxa** [comment]
There's no single place to do it once for resource routes, you would have to add cors to the HTTP server itself, in that case you can use the CORS helpers for your HTTP server.

**Reply by wong2** [reply]
What a pity, I think it's the resource routes that need CORS. Adding CORS to the document route doesn't make much sense, right?
